connectors/src/connectors/confluence/temporal/activities.ts

```diff
@@ -63,24 +58,13 @@ async function fetchConfluenceConnector(connectorId: ModelId) {
 }
 
 async function getConfluenceAccessTokenWithThrow(connectionId: string) {
```

Comment:
I was gonna remove this function that was need while we were phasing out Nango. Now, we can just call `getOAuthConnectionAccessTokenWithThrow` where it's needed.

Reply:
Yeah we can def have one single function for every connector.

Will do in separate PR (unless you're already doing? )